applications/FluidDynamicsHydraulicsApplication/custom_utilities/hydraulic_fluid_auxiliary_utilities.h

```diff
@@ -0,0 +1,151 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Uxue Chasco
+
+#if !defined(KRATOS_HYDRAULIC_FLUID_AUXILIARY_UTILITIES_H)
+#define KRATOS_HYDRAULIC_FLUID_AUXILIARY_UTILITIES_H
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "geometries/geometry.h"
+#include "includes/node.h"
+#include "includes/model_part.h"
+#include "includes/ublas_interface.h"
+#include "modified_shape_functions/modified_shape_functions.h"
+#include "includes/global_pointer_variables.h"
```

Comment:
```suggestion
#include "includes/global_pointer_variables.h"
#include "modified_shape_functions/modified_shape_functions.h"
```
Very minor, it is customary to sort the includes in alphabetical order.